index.js

```diff
@@ -3,7 +3,7 @@ const fs = require('fs');
 const iterm2Version = require('iterm2-version');
 const ansiEscapes = require('ansi-escapes');
 
-class UnsupportedTerminal extends Error {
+class UnsupportedTerminal extends Error { // eslint-disable-line unicorn/custom-error-definition
```

Comment:
Didn't want to change this to `UnsupportedTerminalError` in case some users depends on checking the error name. This would need a major then.